.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
updating php-cs-fixer implies adding this configuration file but also removing the old one .php_cs.dist

https://github.com/LinioPay/http-mock/blob/ccc0a7f4515ce7abe20cfe7682d28b437f6d5dd1/.php_cs.dist#L1

Reply:
I don't remember well but only 1 file is generated when you run php fixer, 

Our gitignore has named both files for result with 2 and with 3

Can you help us here as well

https://github.com/LinioPay/http-mock/blame/ccc0a7f4515ce7abe20cfe7682d28b437f6d5dd1/.gitignore#L7